### PR TITLE
TRUNK-5115-new-6:Replace use of conditional logging with slf4j parame…

### PR DESCRIPTION
**Pull request by gracebish**
…terized messages

Pull request title
TRUNK-5115-new-6:Replace use of conditional logging with parameterized messages
Jira issue number: Jira issue Title
TRUNK-5115:Replace use of conditional logging with parameterized messages
Description of what I changed
Replace use conditionals which are aimed at preventing to construct the debug message with slf4js parametrized messages
Issue I worked on
https://issues.openmrs.org/browse/TRUNK-5115

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by gracebish** [comment]
@dkayiwa . I have made more changes ready for review

File: web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java [hunk 1]
```diff
@@ -9,17 +9,38 @@
  */
 package org.openmrs.web.filter.update;
 
-import liquibase.changelog.ChangeSet;
-import liquibase.exception.LockException;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
```
Comment (dkayiwa) [inline web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java:16]: Can you avoid changing these imports?

Comment (gracebish) [reply web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java:16]: I think it is mvn formating that may have caused the changes in the imports.

Comment (dkayiwa) [reply web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java:16]: Are you sure?

**Comment by gracebish** [comment]
Yes. I am sure.

**Comment by dkayiwa** [comment]
Can you avoid committing them?

File: web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java (hunk 1 above)
Comment (gracebish) [reply web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java:16]: These imports were due to automated formatting tests.

Comment (gracebish) [reply web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java:16]: I am closing this PR to resolve the unwanted imports

**Pull request closed without merging**